package.json

```diff
@@ -80,7 +80,7 @@
     "ip-range-check": "0.0.2",
     "jmespath": "f-w/jmespath.js#semver:^1.0",
     "js-base64": "^3.6.0",
-    "jsonwebtoken": "^8.5.1",
+    "jsonwebtoken": "^9.0.2",
```

Comment:
was there any checks for the changlog with major versions, on how this will affect the current code. ie quick search for any major functionality changes?

Reply:
It's possible to build the image by manually running the GH workflow and then have the tenant repo point to the newly built image. Then we can test out the new image on the dev instance and just see if things are still working.

Reply:
ok, i suspect there should be a way of checking this locally first, we should have this documented, but for now making the assumption we will force the image build, and check it in dev.

Reply:
We can (and should) build the image and test it before merging. There are nice ways to do this with the GH workflow (like to publish a 4.1.12-beta1 version of the image for example) but they aren't currently set up.